test/unit/Fixture/ClassWithProperties.php

```diff
@@ -0,0 +1,13 @@
+<?php
```

Comment:
This file name does not reflect the intended usage - this should be `ClassWithPropertiesAndTraitProperties`

Reply:
Done.